weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallPage.tsx

```diff
@@ -226,6 +249,18 @@ const CallPageInnerVertical: FC<{
             <PaginationControls call={call} path={path} />
           )}
           <Box sx={{marginLeft: showPaginationContols ? 0 : 'auto'}}>
+            {humanFeedbackSpecs && (
```

Comment:
this should probably always show, then the empty state is a CTA/empty state that lets you create / navigates you to the management page